css/index.css

```diff
@@ -1,3 +1,420 @@
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+/* Main Content  */
+.main-content {
+	display: flex;
+	flex-direction: column;
+	border-top: 1px solid black;
+	margin-top: 76px;
+	padding-top: 39px;
+}
+
+.main-content .top-content {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	align-content: center;
+  	padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin-top: 37px;
+	padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 100%;
+  display: inline-block;
+  margin-bottom: 30px;
+  /* margin-right: 12px; */
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+	display: flex;
+	flex-direction: column;
+	align-content: flex-start;
+	margin-top: 57px;
+	border-top: 1px solid black;
+	padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 10px 0;
+}
+
+footer {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin: 32px 0;
+	text-align: center;
+	font-size: 14px;
+}
+
+/*Services Page*/
+
+/*Services CTA*/
+
+.services-cta{
+	display: flex;
+	flex-direction: column;
+	padding-bottom: 80px;
+}
+
+.services-cta img {
+	padding-top: 20px;
+}
+
+.services-cta div {
+	padding-top: 20px;
+}
+
+.services-list{
+	display: flex;
+	flex-direction: column;
+}
+
+.services-row {
+	display:flex;
+	flex-direction: row;
+}
+
+.services-box{
+	display: flex;
+	flex-direction: column;
+	align-content: space-around;
+	align-items: flex-start;
+	padding: 20px;
+	border: 1px solid black;
+	margin: 20px;
+	background-color:#e6ebed;
+}
+
+.learn-more-button{
+	padding: 10px;
+	margin-top: 15px;
+	text-align: center;
+	border: 1px solid black;
+	border-radius: 5px;
+	background-color: white;
+}
+
+.services-facts{
+	display: flex;
+	flex-direction: row;
+}
+
+.services-facts-text{
+	display: flex;
+	flex-direction: column;
+	align-content: space-around;
+	padding: 20px;
+	margin: 20px;
+}
+
+.services-info-img {
+	padding: 20px;
+}
+
+.services-facts-text-component{
+	padding: 5px;
+}
+
+
+/*Media Queries*/
+
+/*Mobile*/
+@media (max-width:500px) {
+
+    body {
+        border: 1px solid gray;
+    }
+    .container {
+        max-width: 90%;
+    }
+
+    header {
+        width:100%;
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    header nav {
```

Comment:
Which of these properties do you think are still in effect from the desktop/tablet view?

>header nav {
  display: flex;
  width: 80%;
  justify-content: space-evenly;   
  margin-top: 35px;